src/iocore/cache/Stripe.h

```diff
@@ -115,7 +117,7 @@ class Stripe
    *
    * @see START_POS
    */
-  Stripe(CacheDisk *disk, off_t blocks, off_t dir_skip, int avg_obj_size = -1);
+  Stripe(CacheDisk *disk, off_t blocks, off_t dir_skip, int avg_obj_size = -1, int fragment_size = -1);
```

Comment:
This could make take the vol config directly, especially if we are likely to add more configuration parameters in the future.

Reply:
On second thought, that may not be great, because each config is used for an entire cache span (volume), so the dependencies would get messier. I think it'd be nice to have a config type for stripes at some point, which could maybe be part of the volume config, but let's leave it as is for now.